source/session.ts

```diff
@@ -398,22 +401,54 @@ export class Session {
    * @return {*}      Decoded data
    */
 
-  private decode(data: any, identityMap: Data = {}): any {
+  private decode(
+    data: any,
+    identityMap: Data = {},
+    decodeDatesAsISO: boolean = false
```

Comment:
I was bout to comment that we could make the third argument a `decodeOptions` to future-proof if we have a need for more options, but seeing as this is assumed to be private - I don't think it matters.